submissions/modding_inspiration/modding_inspiration.md

```diff
@@ -0,0 +1,7 @@
+## Modding Inspiration
+
+I love getting feedback on my mods. Sometimes inspiration for new features are found in the oddest places. Last month, a comment was left on my [Generic Logistic Chest](https://mods.factorio.com/mod/Generic_Logistic_Chest) mod from a German YouTube channel linking to a spotlight of the mod. Not speaking any German myself, I still watched it a bit to see how the reviewer was showcasing the mod.
+
+The quick gist of the mod is that there are "generic" versions of logistic chests, that when placed opens a UI to change it to a normal logistic chest (storage, requester, etc). I saw the reviewer place a few of the chests, then they selected a chest from my UI. They appeared to expect that it would change all the chests. At that time, it only changed the last one placed by the player. It was just how it always worked, it wasn't a conscious decision, it just never occurred to me to let the player place several and then change all of them with one click.
+
+It wasn't a request, it wasn't even mentioned in a language I understood, it was just something I observed when watching someone else interact with my mod. In retrospect though, I can't believe it wasn't in the mod from the beginning.
```

Comment:
I really like the concept of this submission but I feel like its missing a bit at the end here. I think it needs:
- Video of the new behaviour
- Some discussion of implementing this new behaviour, maybe talk about some specific issues you've had in implementing it?